packages/apputils-extension/src/index.ts

```diff
@@ -104,6 +104,21 @@ const palettePlugin: JupyterLabPlugin<ICommandPalette> = {
 
 
 /**
+ * The default layout restorer provider.
+ */
+const settingPlugin: JupyterLabPlugin<ISettingRegistry> = {
+  id: 'jupyter.services.setting-registry',
+  requires: [IStateDB],
+  activate: (app: JupyterLab, state: IStateDB) => {
+    const registry = new SettingRegistry();
```

Comment:
Ah right, good point. I originally did it that way and then I forgot to remove `IStateDB` as a requirement when I switched it.